MDS/Collivery/Model/MdsCollivery.php

```diff
@@ -1,7 +1,7 @@
 <?php
 namespace MDS\Collivery\Model;
 
-use SoapClient; // Use PHP Soap Client
+use Exception;
 use SoapFault;  // Use PHP Soap Fault
```

Comment:
Why is the `SoapFault` being used if there's no SOAP in this class?

Reply:
Removed all refs to SOAP